developer_docs/feature/cross-department-po-receiving.md

```
@@ -0,0 +1,95 @@
+  - `pharmacy_purchase_order_list_for_recieve_with_approval.xhtml`
+  - `pharmacy_wholesale/pharmacy_purchase_order_list_for_recieve.xhtml`
+  - `pharmacy_purchase_order_list_for_recieve_dto.xhtml` (DTO variant)
+- **Out of scope**: `SearchController.createPoTableStore(...)`, which backs
+  Store's own separate PO/GRN flow (external distributor purchases into
+  Store) — unrelated to this request.
+
+## Design
+
+Add one boolean config option:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)